### DISCUSSION TITLE
Deceptive site ahead

### DISCUSSION THREAD

**Discussion by AmanSingh6574** [community/community#50915] [Other Feature Feedback, Questions, & Ideas] [answered]
Deceptive site ahead
Attackers on amansingh6574.github.io may trick
All my projects that I have uploaded on GitHub have been mark deceptive dont know the reason
Pls help

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼 , 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). The Community is unable to assist with matters like this so I'll be locking the post. Bet of luck!
